environment.yml

```diff
@@ -3,5 +3,5 @@ channels:
   - https://repo.mamba.pm/emscripten-forge
   - https://repo.mamba.pm/conda-forge
 dependencies:
-  - ipywidgets=8.0.6
-  - folium=0.14
+  - ipywidgets
```

Comment:
So the old pins don't work anymore?

Could updating to the latest versions work, while still keeping the pins?

Reply:
The idea with the pins was to try to keep the repo in a working state over time, best effort of course :)

Reply:
That would work indeed

Reply:
> keep the repo in a working state over time

Unfortunately that's what's breaking it because some versions may have been nuked from emscripten-forge 😅 but that's not normal behavior